<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordInfo</class>
 <widget class="QWidget" name="RecordInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1732</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#RecordInfo{
background-color: rgb(255, 255, 255);
}
QLabel{
font: 18pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);
}
QToolButton{
font: 18pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
border:2px solid rgb(0, 125, 165);
border-radius:10px;
background-color: rgb(0, 125, 165);
}
QToolButton:disabled{
font: 18pt &quot;微软雅黑&quot;;
color: rgb(180, 180, 180);
border:2px solid rgb(180, 180, 180);
border-radius:10px;
}
QTableWidget{
font: 14pt &quot;微软雅黑&quot;;
gridline-color: rgb(0, 0, 0);
border:1px solid rgb(0, 125, 165);
border-radius:5px;
background-color: rgb(255, 255, 255);
}
QTableWidgetItem{
outline:0px;
border:1px solid rgb(0, 0, 0);
}
QDateTimeEdit{
font: 18pt &quot;微软雅黑&quot;;
border-radius:5px;
border:1px solid rgb(0, 125, 165);
}
QDateTimeEdit::up-button{
top:2px;
right:2px;
width: 30px;
height: 26px;				
}
QDateTimeEdit::down-button{
bottom:2px;
right:2px;
width: 30px;
height: 26px;				
}
QScrollBar:vertical{
width:20px;
background:rgba(0,0,0,0%);
margin:0px,0px,0px,0px;
padding-top:25px;   
padding-bottom:25px;    
}
QScrollBar::handle:vertical{
width:10px;
background:rgba(0,0,0,25%);
background:rgb(0,125,165);
border-radius:5px;
min-height:20px;
}
QScrollBar::handle:vertical:hover{
width:10px;
background:rgba(0,125,165,80%);
border-radius:5px;
min-height:20;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="10,1">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="styleSheet">
      <string notr="true">QTableWidget{
border:1px solid rgb(0,125,165);
border-radius:10px;
}</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,1,0,0,0">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>开始
时间</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEditStart">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>60</height>
          </size>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2020</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
         <property name="displayFormat">
          <string>yyyy/MM/dd hh:mm:ss</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>结束
时间</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEditStop">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="displayFormat">
          <string>yyyy/MM/dd hh:mm:ss</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tBtnManual">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>70</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>190</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>手动调时</string>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tBtnAuto">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>90</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>210</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>自动调时</string>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QToolButton" name="tBtnCheck">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>30</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>查询</string>
         </property>
         <property name="icon">
          <iconset resource="../image.qrc">
           <normaloff>:/Image/dataSearch.png</normaloff>:/Image/dataSearch.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tBtnAllDel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>30</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>删除</string>
         </property>
         <property name="icon">
          <iconset resource="../image.qrc">
           <normaloff>:/Image/dataDel.png</normaloff>:/Image/dataDel.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tBtnExport">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>30</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>导出</string>
         </property>
         <property name="icon">
          <iconset resource="../image.qrc">
           <normaloff>:/Image/dataExport.png</normaloff>:/Image/dataExport.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tBtnPageUp">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>40</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>上翻</string>
         </property>
         <property name="icon">
          <iconset resource="../image.qrc">
           <normaloff>:/Image/arrow-up.png</normaloff>:/Image/arrow-up.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tBtnPageDown">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>80</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>190</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>下翻</string>
         </property>
         <property name="icon">
          <iconset resource="../image.qrc">
           <normaloff>:/Image/arrow-down.png</normaloff>:/Image/arrow-down.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tBtnBack">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>30</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>返回</string>
         </property>
         <property name="icon">
          <iconset resource="../image.qrc">
           <normaloff>:/Image/back.png</normaloff>:/Image/back.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../image.qrc"/>
 </resources>
 <connections/>
</ui>
